tests/rule_based_profiler/parameter_builder/conftest.py

```diff
@@ -0,0 +1,184 @@
+from numbers import Number
+from typing import Dict, Optional
+
+import numpy as np
+import pandas as pd
+import pytest
+import scipy.stats as stats
+
+
+def _generate_distribution_parameters(
```

Comment:
@NathanFarmer How come this is a private method?  Any harm making it public?  Thanks!

Reply:
@alexsherstinsky I see no harm in making these methods public, but for what purpose?

Reply:
@NathanFarmer  For use by other classes/method -- seems like a very useful samples generator. Thanks!